js/react_native/e2e/android/app/src/main/java/com/reactnativeonnxruntimemodule/DataHandlerPackage.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.example.reactnativeonnxruntimemodule;
```

Comment:
change the package name make it consistent with examples from ReactNative offical